### ISSUE TITLE
Keyboard focus is lost when you click "I don't have a SUNet ID"

### ISSUE THREAD

**Issue by jvine** [sul-dlss/sul-requests#489]
(Related to #120 but I didn't reopen that because a lot changed around this code near the end and I don't know at what point this got lost.)

When you are using the keyboard only, and you activate (click) "I don't have a SUNet ID" the focus is no longer visible, and you can only forward click into the top of the form again - if there's a long selection list that can be a very long way round. (For some reason, reverse-tabbing from the bottom up doesn't work until you've gone forward at least one tab.)

The focus should be in the first of the newly-exposed fields - which might be library ID or name, depending on the form.

**Comment by ndushay** [comment]
there is a question as to whether this is actually true ...

**Comment by jvine** [comment]
Magic! it's working now.
